### PR TITLE
Fix/typos docs

### PR DESCRIPTION
**Pull request by jokedurnez**

### CODE REVIEW

File: src/api/PermissionsApi.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ export function getAcl(aclKey :UUID[]) :Promise<*> {
 /**
  * `PATCH /permissions`
  *
- * Updates the Ace for a particular ACL Key, only if the user is the owner of the ACL Key.
+ * Updates the ACL for a particular ACL Key, only if the user is the owner of the ACL Key.
```
Comment (UnsungHero97) [inline src/api/PermissionsApi.js]: I think this is actually correct, though confusing. Please revert.

File: src/api/EntityDataModelApi.js [hunk 2]
```diff
@@ -1726,6 +1726,8 @@ export function getAssociationTypeDetails(associationTypeId :UUID) :Promise<*> {
 /**
  * `GET /edm/association/type/{uuid}/available`
  *
+ * Gets all available assocations for the given EntityType UUID.
```
Comment (UnsungHero97) [inline src/api/EntityDataModelApi.js]: Please change `associations` to `AssociationTypes`.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/api/PermissionsApi.js (hunk 1 above)
Comment (jokedurnez) [reply src/api/PermissionsApi.js]: Done.

File: src/api/EntityDataModelApi.js (hunk 2 above)
Comment (jokedurnez) [reply src/api/EntityDataModelApi.js]: Done.

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**